python/dask_cudf/dask_cudf/backends.py

```diff
@@ -354,7 +353,8 @@ def percentile_cudf(a, q, interpolation="linear"):
             # https://github.com/dask/dask/issues/6864
             result[0] = min(result[0], a.min())
         return result.to_pandas(), n
-    if not np.issubdtype(a.dtype, np.number):
+    if a.dtype.kind not in "iufm":
+        # TODO: Do we want to include timedelta?
```

Comment:
I don't think the original code considered `timedelta` either way, so we have some flexibility here IMO.  Most likely the correct behavior is more like datetime / `M`, which is handled above, but I'd want to take care of that in a separate PR.

I think the way you have it now, which is not to change the behavior, looks good to me.